app/views/welcome/index.html.erb

```diff
@@ -4,13 +4,15 @@
 </div>
 <section class="tutorials col md-col-8">
   <% @tutorials.each do |tutorial| %>
+  <% if !current_user.nil? || tutorial.classroom == false %>
```

Comment:
Add conditional so the user must be logged in and that it is a classroom tutorial